modules/p2/ding_message/ding_message.module

```diff
@@ -278,49 +241,35 @@ function ding_message_check_list_for_updates($uid, $element_id, $mid, $debug = F
     $new_elements_count = 0;
   }
 
-  if ($debug) {
-    drupal_set_message(check_plain('Updating list of type: ' . $type));
-  }
-
-  // Handle the different ding_list types.
-  switch ($type) {
-    case 'ting_object':
-    case 'ding_list_user':
-      // Types not eglible for further processing.
-      break;
-
-    case 'taxonomy_term':
-      module_load_include('inc', 'opensearch', 'open.client');
-      // Get the element value.
-      $value = ding_list_get_list_element_value($element);
-      // Load taxonomy term.
-      $term = taxonomy_term_load($value);
-      // Taxonomy terms updates are a ting search
-      // for materials with the subject = term->name.
-      $result = ting_start_query()
-        ->withFieldFilter(TingSearchCommonFields::SUBJECT, check_plain($term->name))
-        ->withCount(50)
-        ->execute();
+  switch ($element->type) {
+    case 'follow_author':
+    case 'search_query':
+      module_load_include('inc', 'opensearch', 'opensearch.client');
+      // Build a search query, since the url path doesn't
+      // match the query accepted by ting_do_search.
```

Comment:
Why are these 3 lines of code above added

As far as I can tell the Opensearch client is not used in these lines of code so there is no reason to load it. Also the comment does not make sense to me.

Reply:
You're right on every account. I've removed all 3 lines, the comment seems to be leftovers from older code perhaps.